Improved Documentation on how to write a changelog (#3856)
sunpy/sunpy: docs/dev_guide/maintainer_workflow.rst

The changelog will be read by users, so this description should be aimed at SunPy users instead of describing internal changes which are only relevant to the developers.

> Instructions on how to write a changelog is given `here`_

Reviewer: ```suggestion
Instructions on how to write a changelog can be found `here`_
```

The changelog will be read by users, so this description should be aimed at SunPy users instead of describing internal changes which are only relevant to the developers.

> Instructions on how to write a changelog can be found `here`_

Reviewer 2: Now that we are with this... we shouldn't use `here` on any of our links!
Read why ['click here' links are not good for accessibility on this Berkeley guide](https://webaccess.berkeley.edu/ask-pecan/click-here).

Reviewer 3: How about now?

The changelog will be read by users, so this description should be aimed at SunPy users instead of describing internal changes which are only relevant to the developers.

The current changelog is kept in the file "CHANGELOG.rst" at the root of the repository.
You do not need to update this file as we use `towncrier`_ to update our changelog.
This is built and embedded into our documentation.

Towncrier will automatically reflow your text, so it will work best if you stick to a single paragraph, but multiple sentences and links are OK and encouraged.
You can install towncrier and then run `towncrier --draft` if you want to get a preview of how your change will look in the final release notes.

> Instructions on how to write a changelog can be found `here`_.

Reviewer 3: ```suggestion
`Instructions on how to write a changelog. <https://github.com/sunpy/sunpy/blob/master/changelog/README.rst>`__.
```

.. _towncrier: https://pypi.org/project/towncrier/

> .. _here: https://github.com/sunpy/sunpy/blob/master/changelog/README.rst

Reviewer 3: ```suggestion
```
